site/content/entry/citra-vulkan-announcement/index.md

```diff
@@ -0,0 +1,110 @@
++++
+date = "2023-08-26T17:00:00+01:00"
+title = "The Vulkan-o erupts - Citra Vulkan is here!"
+tags = [ "feature-update" ]
+author = "autumnburra"
+coauthor = "sleepingsnake"
++++
+
+Hey there, Citra fans! (We haven't used that one for a while, have we?)
+You've all wanted it and we've heard your cries and pleas. Vulkan has now arrived and is here to stay!
+
+# Why Vulkan?
+
+Since the dawn of time- well, since Citra's creation, many, *many* users have requested we add support for the [Vulkan Graphics API](https://en.wikipedia.org/wiki/Vulkan.); better performance in many cases, better support on devices, especially Android. What is there not to love about it?
```

Comment:
```suggestion
Since the dawn of time — well, since Citra's creation — many, *many* users have requested we add support for the [Vulkan Graphics API](https://en.wikipedia.org/wiki/Vulkan.); providing better performance in many cases and better support on devices, especially Android. There's a lot to love, but oh boy, was it ever a challenge!
```